pkg/container/qemu_runner.go

```diff
@@ -550,29 +550,29 @@ func createMicroVM(ctx context.Context, cfg *Config) error {
 		}
 	}
 
+	// default to use 85% of available memory, if a mem limit is set, respect it.
+	mem := int64(float64(getAvailableMemoryKB())*0.85)
 	if cfg.Memory != "" {
 		memKb, err := convertHumanToKB(cfg.Memory)
 		if err != nil {
 			return err
 		}
 
-		if memKb > int64(getAvailableMemoryKB()) {
-			log.Warnf("qemu: requested too much memory, requested: %d, have: %d", memKb, getAvailableMemoryKB())
```

Comment:
Should we keep a warning when the build wants more resources than folks have?

Reply:
This shouldn't happen
we default to 85% ram, and we use cfg.Memory only if it's lower than that